src/prelude.rs

```diff
@@ -1,2 +1,3 @@
 //! The Tide prelude.
 pub use http_types::Status;
+pub use serde_json::json;
```

Comment:
would be nice to make `serde_json` a feature dependency, even if enabled by default

Reply:
I expected to need to do that when starting this pr, but discovered it was already a normal dependency. We'd probably want to gate json_body on this as well

Reply:
Yep, that would be a good improvement, given that serde_json is not the lightest dependency

Reply:
Added a "json" feature, enabled by default, and gated everything json-related in tide on it. Probably could stand to do the same for http-types as well